addons/pos_self_order/tests/self_order_common_test.py

```diff
@@ -3,15 +3,54 @@
 
 import odoo.tests
 
+from odoo import Command
+
 
 @odoo.tests.tagged("post_install", "-at_install")
 class SelfOrderCommonTest(odoo.tests.HttpCase):
     browser_size = "375x667"
     touch_enabled = True
 
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.pos_manager = cls.env['res.users'].create({
+            'name': 'POS Manager',
+            'login': 'pos_manager',
+            'password': 'pos_manager',
+            'groups_id': [
+                (4, cls.env.ref('base.group_user').id),
+                (4, cls.env.ref('base.group_erp_manager').id),
+                (4, cls.env.ref('base.group_partner_manager').id),
+                (4, cls.env.ref('account.group_account_manager').id),
+                (4, cls.env.ref('point_of_sale.group_pos_manager').id),
+            ],
+        })
```

Comment:
We already have a `pos_admin` user who comes from `TestPointOfSaleHttpCommon` I don't know if this is a good way of inertizing this class.

Reply:
This test is not inheriting `TestPointOfSaleHttpCommon`. Let's not inherit from it for this fix because it can carry info that might interfere with the existing tour. It will probably a lot of work to adjust the tour in order to inherit from the said common test class.